pinning/hardware/netio.m4

```diff
@@ -1,3 +1,6 @@
+/* add support for RS485 */
+pin(RS485TE_USART0, PD2, OUTPUT)
```

Comment:
PD2 is already used by multiple other modules. Please enclose this definition in a conditional.